src/common/dcsvg.cpp

```diff
@@ -224,50 +270,79 @@ wxString wxGetBrushStyleName(wxBrush& brush)
     return brushStyle;
 }
 
-wxString wxGetBrushFill(wxBrush& brush)
+wxString wxGetBrushFill(wxBrush const& brush)
```

Comment:
As much as I prefer east const myself, I think we should stick to west const in wx code for consistency:
```suggestion
wxString wxGetBrushFill(const wxBrush& brush)
```

Reply:
Fixed all occurrences.